src/applications/disability-benefits/all-claims/utils/submit.js

```diff
@@ -223,7 +225,8 @@ export function getPtsdChangeText(changeFields = {}) {
 }
 
 export function filterServicePeriods(formData) {
-  const { serviceInformation } = formData;
+  // migrate branches in case prefill migration is skipped somehow
+  const { serviceInformation } = migrateBranches(formData);
```

Comment:
@Mottie I forget whether we went through this before, but, is it an issue here that `migrateBranches` doesn't mutate `formData`?

Reply:
We shouldn't be mutating `formData` within the submit transformer, all of these functions should be... we're building the data to match what is actually submitted to the backend - so yeah maybe we do need to clone it before making these changes.

Reply:
@Mottie Or I guess, my question is: is it your intent to transform the names of the branches within `filterServicePeriods`, so that `clonedData` contains the new branch names? If it isn't then we can leave the code as-is.